### DISCUSSION TITLE
JSON format Display

### DISCUSSION THREAD

**Discussion by bllackeye** [pulumi/pulumi#7656] [Pulumi in Practice] [answered]
Pulumi Preview supports JSON format Display. 

Is there any reasons that  Pulumi up/destroy doesn't support JSON format Display? I feel like it is very useful if one wants to keep the Display as a log.

**Comment by nimbinatus** [comment, marked as answer]
Mainly, the issue is that JSON is only printed at the end for `preview`. If you're trying to run a longer program, streaming JSON is better, and we're still working on it. Here's the current issue that discusses this request for more info: https://github.com/pulumi/pulumi/issues/2390
